File: carsharing-app/app/src/main/java/com/syndicate/carsharing/database/models/Penalty.kt
package com.syndicate.carsharing.database.models

import com.fasterxml.jackson.annotation.JsonClassDescription
import com.fasterxml.jackson.annotation.JsonProperty

data class Penalty (
    @JsonProperty("Id") val id: Int,
    @JsonProperty("UserId") val userId: Int,
    @JsonProperty("RelatedRent") val relatedRent: Int,
    @JsonProperty("RatingPenalty") val ratingPenalty: Int,
    @JsonProperty("Price") val price: Double,
    @JsonProperty("Description") val description: String,
    @JsonProperty("IsPaid") val isPaid: Boolean,
)

File: carsharing-api/carsharing-api/Entities/Penalty.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace carsharing_api.Entities;

[Table("penalty")]
public class Penalty
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public double Price { get; set; }
    public bool IsPaid { get; set; }
    public int RelatedRent { get; set; }
    public string Description { get; set; }
    public int RatingPenalty { get; set; }
}